ee/clickhouse/sql/dead_letter_queue.py

```diff
@@ -46,7 +46,12 @@
     ttl_period=ttl_period("_timestamp", 4),  # 4 weeks
 )
 
-KAFKA_DEAD_LETTER_QUEUE_TABLE_SQL = lambda: DEAD_LETTER_QUEUE_TABLE_BASE_SQL.format(
+# skip up to 1000 messages per block. blocks can be as large as 65505
+# if a block has >1000 broken messages it probably means we're doing something wrong
```

Comment:
or someone could be DDOS'ing us? Is there a graph that will show broken messages if we skip them?

Reply:
well in that case we should be made aware too